instance/.env.example

```
@@ -26,6 +26,15 @@ SUPABASE_DB_NAME=postgres
 
 SUPABASE_REST_API=False
 
+MONGO_SCHEME=mongodb
+MONGO_HOST=localhost
+MONGO_DB="devdox"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)